zyte_common_items/components.py

```diff
@@ -426,6 +426,17 @@ class Video(_Media):
     """
 
 
+def cast_request(value: Request, cls: Type[Request]) -> Request:
+    new_value = copy_attributes(value, cls)
```

Comment:
Is it always required to make a copy?

Reply:
No, we can skip this if the classes are the same. We don't do that for Metadata though, I guess we can add that there as well.